src/testRunner/unittests/base64.ts

```diff
@@ -10,7 +10,7 @@ describe("unittests:: base64", () => {
                 "日本語",
                 "🐱",
                 "\x00\x01",
-                "\t\n\r\\\"\'\u0062",
+                "\t\n\r\\\"'\u0062",

```

Comment:
On second look, I'm not sure if this intended to write `'` or `\\'`. But, I'm not sure what "escaping" there is to check when roundtripping base64 anyway...